ui/src/app/views/profile-page/component.html

```diff
@@ -106,19 +144,19 @@ <h6 class="profileCompletionText">
       </div>
       <div>
         <button class="btn btn-secondary" (click)="submitTermsOfService()"
-            [disabled]="email === null || this.profile.termsOfServiceCompletionTime !== null">
+            [disabled]="email === null || this.workingProfile.termsOfServiceCompletionTime !== null">
           Submit Terms of Service
         </button>
       </div>
       <div>
         <button class="btn btn-secondary" (click)="completeEthicsTraining()"
-            [disabled]="email === null || this.profile.ethicsTrainingCompletionTime !== null">
+            [disabled]="email === null || this.workingProfile.ethicsTrainingCompletionTime !== null">
           Complete Ethics Training
         </button>
       </div>
       <div>
         <button class="btn btn-secondary" (click)="submitDemographicSurvey()"
-            [disabled]="email === null || this.profile.demographicSurveyCompletionTime !== null">
+            [disabled]="email === null || this.workingProfile.demographicSurveyCompletionTime !== null">
```

Comment:
If these don't need to be checked against null specifically you can simplify the condition to `email && this.workingProfile.demographicsSurveyCompletionTime` and similarly for the others above.